javascript/reactadmin/corebosServerProvider.ts

```diff
@@ -241,7 +238,7 @@ const execQuery = (resource: string, quesrParams: any, additionalWhereClause?: a
     }
 
     query = encodeURIComponent(query);
-    return doQueryWithTotal(query).then((data: any) => {
+    return cbconn.doQueryWithTotal(query).then((data: any) => {
```

Comment:
use correct data instead of any

Reply:
Hello @rkoleci 
I didnt want to add expected/correct data type because I worry that the response might change, so better to leave it to the developer who who will call that method to define the interfaces in his/her application.
Now the expected interface is this 
`{result: any[] , totalrows: number }`
Again, I have to define also the interface for result value instead of using `any` data type and I have to do so for all modules considered the changes are made in fields in coreBOS very often.

Reply:
You can add a T type

data: T[]